index.html

```diff
@@ -1,53 +1,104 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="description" content="homework" />
-    <meta name="keywords" content="lists tables and forms" />
-    <meta name="author" content="Alexey Tretyakov" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./css/style.css" />
-    <meta charset="UTF-8" />
-    <title>lesson</title>
-  </head>
-  <body>
-    <section class="advantages">
-      <h2 class="heading">What do we offer</h2>
-      <hr class="highlight" />
-      <div class="content">
-        <div class="vectors">
-          <img
-            class="vectors-img"
-            src="./img/headphones.png"
-            alt="Support 24/7"
-          />
-          <p class="img-text-v">Support 24/7</p>
-        </div>
-        <div class="vectors">
-          <img
-            class="vectors-img"
-            src="./img/communicate.png"
-            alt="Communicate directly"
-          />
-          <p class="img-text-v">Communicate directly</p>
-        </div>
-        <div class="vectors">
-          <img
-            class="vectors-img"
-            src="./img/bookonline.png"
-            alt="Book online"
-          />
-          <p class="img-text-v">Book online</p>
-        </div>
-        <div class="vectors">
-          <img
-            class="vectors-img"
-            src="./img/like.png"
-            alt="Real quest reviews"
-          />
-          <p class="img-text-v">Real quest reviews</p>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="description" content="homework" />
+  <meta name="keywords" content="lists tables and forms" />
+  <meta name="author" content="Alexey Tretyakov" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/style.css" />
+  <meta charset="UTF-8" />
+  <title>lesson</title>
+</head>
+<body>
+<section >
+  <div class="content">
+    <div class="block-1">
+      <img class="logo_vector" src="./img/logo_vector.svg" alt="logo">
+      <div class="menu">
```

Comment:
I recommend to use `<nav>` instead of `<div>`
Example of the best practice 
```
<nav>
  <ul>
    <li aria-current="page">
      <span><a href="#">Шапки, шарфы, варежки</a></span>
    </li>
    <li><a href="#">Цены</a></li>
    <li><a href="#">Контакты</a></li>
  </ul>
</nav>
```

[More info](https://weblind.ru/inner.html#nav-menu)